src/pages/teachers/index.tsx

```diff
@@ -43,51 +44,54 @@ const Teachers: NextPage<TeachersHomePageProps> = (props) => {
   });
 
   return (
-    <AppLayout seoProps={BETA_SEO_PROPS} $background={"white"}>
-      <HomePageTabImageNav current={current} setCurrent={setCurrent} />
-      {current === "teachers" && (
-        <TeachersTab keyStages={curriculumData.keyStages} />
-      )}
-      {current === "curriculum" && <CurriculumTab />}
-      {current === "pupils" && <PupilTab />}
-      <MaxWidth $mv={[24, 56]}>
-        <Box $ph={[16, 24]} $height={"100%"}>
-          <Flex
-            $width={"100%"}
-            $alignItems={["flex-start", "center"]}
-            $justifyContent="space-between"
-            $mb={48}
-            $flexDirection={["column", "row"]}
-          >
-            <Heading $mb={[24, 0]} tag={"h2"} $font={"heading-5"}>
-              Stay up to date
-            </Heading>
-            <Flex $flexDirection={"row"}>
-              <Typography $mr={16} $font="heading-7">
-                <OakLink page={"webinar-index"}>All webinars</OakLink>
-              </Typography>
-              <Typography $font="heading-7">
-                <OakLink page={"blog-index"}>All blogs</OakLink>
-              </Typography>
+    <>
+      <HomePageBanner />
```

Comment:
should this live in the `AppHeader`? seems a bit strange just floating above `AppLayout`